lib/ui/pointer.dart

```diff
@@ -380,6 +381,23 @@ class PointerData {
   /// The current angle of the pan/zoom in radians, with 0.0 as the initial angle.
   final double rotation;
 
+  // An optional function that allows the framework to respond to the event
+  // that triggered this PointerData instance.
+  final void Function({bool allowPlatformDefault})? _respond;
+
+  /// Function that the framework/app can call to respond to the native event
+  /// that triggered this PointerData.
+  ///
+  /// The parameter [allowPlatformDefault] allows the platform to perform the
+  /// default action associated to the native event when it's set to `true`.
+  /// The default value depends on the embedder, for example in the web it is
```

Comment:
I don't understand what's meant by "default value" here since the parameter is required and doesn't have a default.

Reply:
Removed the bit about the default value.